content/en/platform/corda/5.0/deploying-operating/observability/metrics.md

```diff
@@ -230,3 +235,54 @@ the maximum value, and the cumulative sum of values, respectively.
 In addition, the Corda metrics endpoint also includes Caffeine cache metrics (`corda_cache_*`),
 Kafka producer and consumer client metrics (`corda_kafka_*`), JVM metrics (`jvm_*`), process metrics (`process_*`),
 and system metrics (`system_*`) provided by the corresponding Micrometer bindings.
+
+#### Database Worker
+
+The database worker is the sole worker with access to the database, apart from the crypto worker, which has its
+own dedicated database. As a result, the activities carried out within the database worker pertain exclusively to the database.
+
+The database worker is responsible for handling and serving persistence requests originating from various Corda worker types,
+such as the flow worker or the MGM worker. The flow persistence requests metrics presented in this section measure:
+
+* The time needed to handle the flow persistence requests.
+* The time spent by the flow persistence requests on Kafka from the moment they were added by flows until they were received
```

Comment:
Could it be: "_The time the flow persistence requests remained on Kafka from..._"

Reply:
Updated according to the suggestion.